src/Update.elm

```diff
@@ -0,0 +1,25 @@
+module Update (update) where
+
+import Array exposing (toList, fromList)
+import Char exposing (fromCode)
```

Comment:
It looks like these are no longer being used, so you can delete them. If you turn on compiler warnings with in `elm make` with the `-w` switch, it'll actually tell you about stuff like this, which is cool.